packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/workflow/processes/GetProcessSanityTests.java

```diff
@@ -59,4 +60,13 @@ public void getProcessByAdmin() throws Exception
         process.assertThat().field("id").is(addedProcess.getId())
                 .and().field("startUserId").is(addedProcess.getStartUserId());
     }
+
+    @TestRail(section = { TestGroup.REST_API, TestGroup.WORKFLOW,TestGroup.PROCESSES }, executionType = ExecutionType.SANITY,
```

Comment:
This should probably have `section = { TestGroup.REST_API, TestGroup.PROCESSES }`, to be conform to the other tests.

Reply:
Fixed.